src/calc.py

```diff
@@ -9,12 +9,13 @@ def add(a, b, third_operand = 0):
     """
     return a + b + third_operand
 
-def sub(a, b):
+def sub(a, b, c = 0):
```

Comment:
Single letter variable names are not great. Try this instead
```suggestion
def sub(a, b, third_operand = 0):
```